lib/services/dependency-extensions-service-base.ts

```diff
@@ -36,6 +37,7 @@ export class DependencyExtensionsServiceBase extends serverExtensionsBaseLib.Ext
 
 				this.$progressIndicator.showProgressIndicator(this.prepareExtensionBase(dependencyExtensionData, cachedVersion), 5000).wait();
 				this.$progressIndicator.showProgressIndicator(afterPrepareAction(), 100).wait();
+				this.$logger.printInfoMessageOnSameLine(os.EOL);
```

Comment:
We've discussed this in personal and we'll improve all these progressIndicator and printInfo... methods in a separate PR